openedx/features/colaraz_features/api/views.py

```diff
@@ -22,18 +24,33 @@ class SiteOrgViewSet(viewsets.ViewSet):
     """
     View set to enable creation of site, organization and theme via REST API.
     """
+    authentication_classes = (OAuth2Authentication,)
+    permission_classes = (IsAuthenticated, IsAdminUser)
     serializer_class = serializers.SiteOrgSerializer
 
     def create(self, request):
         """
         Perform creation operation for site, organization, site theme and site configuration.
         """
-        serializer = self.serializer_class(data=request.POST)
+        try:
+            serializer = self.serializer_class(data=request.data)
+            if serializer.is_valid(raise_exception=True):
+                data = serializer.create(serializer.validated_data)
+        except (rest_serializers.ValidationError, ValueError, NameError) as ex:
+            return Response(
+                {'error': str(ex)},
+                status=status.HTTP_400_BAD_REQUEST
+            )
+        except:
```

Comment:
```
openedx/features/colaraz_features/api/views.py:44:9: E722 do not use bare 'except'
```

Reply:
It's the second except for unpredictable errors.